code/src/components/album.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import '../index.css';
+import { ArtistsName } from './ArtistsName';
+import { AlbumImage } from './AlbumImage';
+
+const Album = ({ name, artists, image, link }) => {
+  console.log('album', name, artists);
+  return (
+    <section className="album">
+      <div><AlbumImage image={image} name={name} link={link} /></div>
+      <div className="textContainer">
+        <div className="albumDetails">
+          <a href={link} target="_blank" rel="noreferrer" className="albumLink">
+            <span>{name}</span>
+          </a>
+        </div>
+        <div className="artistDetails">
+          {artists.map((artist, index) => {
+            const isLast = index === artists.length - 1;
+            return (
+              <ArtistsName
+                key={artist.id}
+                name={artist.name}
+                isLast={isLast}
+                artistUrl={artist.external_urls.spotify} />
+            );
+          })}
+        </div>
+      </div>
+    </section>
+  );
+};
+
```

Comment:
Nice work on the variable ```isLast```. But if there were three artists, would there be a ```,```between the first and second name?

Reply:
That was a stretch goal, so we decided to do a simpler version. It's a mix between having a comma between every artist and the stretch goal version. Since there were only two artists to try out, we skipped the stretch goal. We just thought it looked nicer with &.